code/game/machinery/stasis.dm

```diff
@@ -127,7 +127,7 @@
 	update_icon()
 
 /obj/machinery/stasis/proc/check_patient()
```

Comment:
what kind of shittery is this proc, just replace any uses of this with `if(ishuman(occupant) ||  ismonkey(occupant))`

Reply:
yep literally only /obj/machinery/computer/operating/ui_data(mob/user) uses this

Reply:
Yeah, you're right